python/cuml/dask/common/comms.py

```diff
@@ -114,7 +114,8 @@ def _func_ucp_listener_port(sessionId, r):
     return worker_state(sessionId)["ucp_listener"].port
 
 
-async def _func_init_all(sessionId, uniqueId, comms_p2p, worker_info, verbose):
+async def _func_init_all(sessionId, uniqueId, comms_p2p,
+                         worker_info, verbose, streams_per_handle):
```

Comment:
I don't think so. I don't include defaults in the functions on the workers because I want error if I didn't pass all the params. This function isn't really reusable anyways.